mne/coreg.py

```diff
@@ -1206,3 +1206,67 @@ def _scale_xfm(subject_to, xfm_fname, mri_name, subject_from, scale,
                 F_mri_ras, 'ras', 'ras'),
             F_ras_mni, 'ras', 'mni_tal')
     _write_fs_xfm(fname_to, T_ras_mni['trans'], kind)
+
+
+def get_mni_fiducials(subject, subjects_dir):
+    """Estimate fiducials for a subject.
+
+    Parameters
+    ----------
+    subject : str
+        Name of the mri subject
+    subjects_dir : None | str
+        Override the SUBJECTS_DIR environment variable
+        (sys.environ['SUBJECTS_DIR'])
+
+    Returns
+    -------
+    fids_mri : list
+        List of estimated fiducials (each point in a dict)
+    """
+    from .transforms import _read_fs_xfm
+    from .io.tests import data_dir as fsaverage_dir
+    if not has_nibabel():
+        raise ImportError('This function requires nibabel.')
+        return
+    from nibabel import freesurfer as fs
+
+    subjects_dir = op.join(
+            get_subjects_dir(subjects_dir, raise_error=True), subject)
+    fname_orig = op.join(subjects_dir, 'mri', 'T1.mgz')
+    fname_tal = op.join(subjects_dir, 'mri', 'transforms', 'talairach.xfm')
+    fname_fids_fs = op.join(fsaverage_dir, 'fsaverage-fiducials.fif')
+
+    if not op.isfile(fname_fids_fs):
+        raise IOError('This function requires the MNE testing dataset')
+        return
+    fids_default, coord_frame = read_fiducials(fname_fids_fs)
+    xfm_tal = np.matrix(_read_fs_xfm(fname_tal)[0])
+
+    # Get Freesurfer vox2ras and vox2ras-tkr as matrices
```

Comment:
This comment just states what we can read in the code before. 

Instead of this, it would be nice if you could give some rationale / background about the different coordinate systems instead. When reading the code, that's what I usually struggle to understand as these are not so well-documented.

Reply:
I think I used https://surfer.nmr.mgh.harvard.edu/fswiki/CoordinateSystems as my reference for the transformation stuff.  There's also good content on this in the plot_source_alignment tutorial.  Would a pointer to these be useful, or are you thinking that it would be more helpful to include a condensed version of some of that content?

Reply:
We do need to overhaul coord frame docs, there is an issue about it. It's particularly critical in two places: 1) vol source space, and 2) vol morphing. So I'd rather tackle it in a general overhaul rather than here.

But I will say that we ship the fs fids with MNE so you should just use that and not rely on them being in the `subjects_dir` that the user passes:

https://github.com/mne-tools/mne-python/tree/master/mne/data/fsaverage

Reply:
I can include a link to the transformation details for the curious, and leave out the specifics for now.

I think the fsaverage fids are coming from the testing data right now, not from `subjects_dir` (i.e., `from .io.tests import data_dir as fsaverage_dir` & `fname_fids_fs = op.join(fsaverage_dir, 'fsaverage-fiducials.fif')` -- in the function, `subjects_dir` is only used to get the subject's transformations.  It would definitely make more sense to get them from the location you provided, rather than from the testing data.